internal/hcsoci/resources_wcow.go

```diff
@@ -19,6 +19,10 @@ import (
 )
 
 func allocateWindowsResources(coi *createOptionsInternal, resources *Resources) error {
+	if coi.Spec == nil || coi.Spec.Windows == nil || coi.Spec.Windows.LayerFolders == nil {
+		return fmt.Errorf("layerforlders not populated")
```

Comment:
Should we say "spec.Windows.LayerFolders not populated". I dont imagine a conflict but its very clear this way.